packages/vega-typings/types/spec/scale.d.ts

```diff
@@ -78,21 +79,23 @@ export type ScaleInterpolate =
       type: 'rgb' | 'cubehelix' | 'cubehelix-long' | SignalRef;
       gamma?: number | SignalRef;
     };
-export interface DataRef {
+export interface ScaleDataRef {
   data: string;
   field: ScaleField;
 }
-export type MultiDataRef =
-  | {
-      data: string;
-      fields: ScaleField[];
-    }
-  | {
-      fields: ((string | number | boolean)[] | DataRef | SignalRef)[];
-    };
-export type ScaleData =
-  | (DataRef & { sort?: SortField })
-  | (MultiDataRef & { sort?: UnionSortField });
+
+export type ScaleMultiDataRef = {
```

Comment:
Why not multi scale data ref?

Reply:
It's single scale, not multi scale?  I'm also just treating scale as prefix here.